### PR TITLE
r10k: Fix broken/outdated links and remove trailing whitespace

### PR DESCRIPTION
**Pull request by elyscape**

### CODE REVIEW

File: source/pe/3.8/r10k_reference.md [hunk 1]
```diff
@@ -48,4 +48,4 @@ The `r10k puppetfile` command accepts the following subcommands:
 
 * `check`: Verifies the Puppetfile syntax is correct.
 * `install`: Installs all modules from a Puppetfile.
-* `purge`: Purges unmanaged modules from a Puppetfile managed directory.
```
Comment (adrienthebo) [inline source/pe/3.8/r10k_reference.md]: Why was this command omitted?

Comment (elyscape) [inline source/pe/3.8/r10k_reference.md]: Huh, weird. Lemme fix that.

Comment (elyscape) [inline source/pe/3.8/r10k_reference.md]: So the weird thing is that it wasn't.  Something odd appears to have happened in the GitHub interface.

Comment (elyscape) [inline source/pe/3.8/r10k_reference.md]: Yeah, c.f. the [patch view](https://patch-diff.githubusercontent.com/raw/puppetlabs/puppet-docs/pull/496.patch):
```patch
diff --git a/source/pe/3.8/r10k_reference.md b/source/pe/3.8/r10k_reference.md
index d9eaf5c..73e7631 100644
--- a/source/pe/3.8/r10k_reference.md
+++ b/source/pe/3.8/r10k_reference.md
@@ -28,8 +28,8 @@ Each subcommand allows the following options:
 * `-h`, `--help`: Shows help for this command.
 * `-t`, `--trace`: Displays stack traces on application crash.
 * `-v`, `--verbose`: Sets log verbosity. Valid values: fatal, error, warn, notice, info, debug, debug1, debug2.
- 
- 
+
+
 ###`deploy`
 
 The `r10k deploy` command accepts the following subcommands:
@@ -37,7 +37,7 @@ The `r10k deploy` command accepts the following subcommands:
 * `display`: Displays a list of environments in your deployment.
   * To display a list of modules in the Puppetfile, add the `-p` flag to the command.
   * To display detailed information, add `--detail` to the command.
-* `environment`: Deploys environments and their dependent modules. 
+* `environment`: Deploys environments and their dependent modules.
   * To update modules specified in the environment's Puppetfile, add the `-p` flag to the command.
 * `module`: Deploys a module in all environments.
   * To update the modules in the environment, add the `-e` flag to the command.
@@ -48,4 +48,4 @@ The `r10k puppetfile` command accepts the following subcommands:
 
 * `check`: Verifies the Puppetfile syntax is correct.
 * `install`: Installs all modules from a Puppetfile.
-* `purge`: Purges unmanaged modules from a Puppetfile managed directory.
\ No newline at end of file
+* `purge`: Purges unmanaged modules from a Puppetfile managed directory.
```

Comment (jbondpdx) [inline source/pe/3.8/r10k_reference.md]: Yeah, I didn't see it in the PR either. Oh, Github....
Anyway, I'm embarrassed that I had these errors, but thank you for catching them, @elyscape !

File: source/pe/3.8/r10k_setup.md [hunk 2]
```diff
@@ -102,4 +103,4 @@ Both /etc/puppetlabs/r10k/r10k.yaml and /etc/r10k.yaml configuration files exist
 
 ## Next Steps
 
-After you've prepared r10k, you're ready to configure [r10k.yaml][r10kyaml] and Puppet to use directory environments.
\ No newline at end of file
+After you've prepared r10k, you're ready to configure [r10k.yaml][r10kyaml] and Puppet to use directory environments.
```
Comment (elyscape) [inline source/pe/3.8/r10k_setup.md]: Just as a note, this wasn't removed either:
```patch
diff --git a/source/pe/3.8/r10k_setup.md b/source/pe/3.8/r10k_setup.md
index 1a3f77d..87f6661 100644
--- a/source/pe/3.8/r10k_setup.md
+++ b/source/pe/3.8/r10k_setup.md
[snip]
@@ -102,4 +103,4 @@ Both /etc/puppetlabs/r10k/r10k.yaml and /etc/r10k.yaml configuration files exist
 
 ## Next Steps
 
-After you've prepared r10k, you're ready to configure [r10k.yaml][r10kyaml] and Puppet to use directory environments.
\ No newline at end of file
+After you've prepared r10k, you're ready to configure [r10k.yaml][r10kyaml] and Puppet to use directory environments.
```

**Pull request merged**